diff --git a/CMakeLists.txt b/CMakeLists.txt
index eadd4ec..48746e9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,7 +34,7 @@ add_definitions(-D_LIBRYZENADJ_INTERNAL)
 ADD_EXECUTABLE(${PROJECT_NAME} ${OS_SOURCE} ${COMMON_SOURCES} argparse.c main.c)
 target_link_libraries(${PROJECT_NAME} ${OS_LINK_LIBRARY})
 #SET_TARGET_PROPERTIES(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE C)
-ADD_LIBRARY (libryzenadj SHARED ${OS_SOURCE} ${COMMON_SOURCES})
+ADD_LIBRARY (libryzenadj ${OS_SOURCE} ${COMMON_SOURCES})
 set_target_properties(libryzenadj PROPERTIES PREFIX "")
 target_link_libraries(libryzenadj ${OS_LINK_LIBRARY})
 #SET_TARGET_PROPERTIES(libryzenadj PROPERTIES LINKER_LANGUAGE C)
